src/API/WesternStatesWater.WestDaat.Client.Functions/ConservationFunction.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using System.Net;
+using WesternStatesWater.WestDaat.Contracts.Client;
+using WesternStatesWater.WestDaat.Contracts.Client.Requests.Conservation;
+using WesternStatesWater.WestDaat.Contracts.Client.Responses.Conservation;
+
+namespace WesternStatesWater.WestDaat.Client.Functions;
+
+public class ConservationFunction : FunctionBase
+{
+    private readonly IApplicationManager _applicationManager;
+    private readonly ILogger _logger;
+
+    private const string RouteBase = "conservation";
+
+    public ConservationFunction(IApplicationManager applicationManager, ILogger<ConservationFunction> logger)
+    {
+        _applicationManager = applicationManager;
+        _logger = logger;
+    }
+
+    [Function(nameof(EstimateEvapotranspiration))]
+    [OpenApiOperation(nameof(EstimateEvapotranspiration))]
+    [OpenApiResponseWithBody(HttpStatusCode.OK, "OK", typeof(EstimateEvapotranspirationResponse))]
+    public async Task<HttpResponseData> EstimateEvapotranspiration(
+        [HttpTrigger(AuthorizationLevel.Function, "get", Route = $"{RouteBase}/estimate")]
```

Comment:
```suggestion
        [HttpTrigger(AuthorizationLevel.Function, "post", Route = $"{RouteBase}/estimate")]
```

Should be post here, right?

Reply:
yes, thank you. updating